components/dialog/test/dialog-fullscreen.vdiff.js

```diff
@@ -0,0 +1,79 @@
+import '../dialog-fullscreen.js';
+import { expect, fixture, html, nextFrame } from '@brightspace-ui/testing';
+import { footer, general, long, tabs } from './dialog-shared-contents.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+
+function dialog(opts) {
+	const defaults = { content: html`${general}${footer}`, noPadding: false };
+	const { content, noPadding, width } =  { ...defaults, ...opts };
+	return html`
+		<d2l-dialog-fullscreen title-text="Dialog Title" ?no-padding="${noPadding}" width="${ifDefined(width)}" opened>
+			${content}
+		</d2l-dialog-fullscreen>
+	`;
+}
+
+function dispatchFullscreenWithinEvent(elem, state) {
+	elem.dispatchEvent(new CustomEvent(
+		'd2l-fullscreen-within', { bubbles: true, composed: true, detail: { state: state } }
+	));
+}
+
+describe('dialog-fullscreen', () => {
+
+	['native', 'custom'].forEach((type) => {
```

Comment:
To note: We currently have `native` dialogs off because of a number of defects.  So we actually have duplicates of every vdiff test, with both `native` and `custom` using `custom` dialogs.  We can leave this running both (I guess to verify that native is properly turned off), or we could just comment out the `'native'` option until it does something 🤷

Reply:
Yeah I was thinking about this the other day when you mentioned migrating these tests. I'd be all for commenting out the native tests. We can reenable the tests to generate goldens before we reenable the native implementation so we have useful diffs for that change.